webapp-django/crashstats/crashstats/admin.py

```diff
@@ -174,6 +175,8 @@ def process_crashes(modeladmin, request, queryset):
 
 @admin.register(MissingProcessedCrashes)
 class MissingProcessedCrashesAdmin(admin.ModelAdmin):
+    """DEPRECATED."""
```

Comment:
I'll take the deprecated things out in a later PR after everything has been pushed to prod.